OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -2648,19 +2648,52 @@ public static DreamValue NativeProc_sorttext(NativeProc.State state)
         [DreamProc("text2path")]
         [DreamProcParameter("T", Type = DreamValueType.String)]
         public static DreamValue NativeProc_text2path(NativeProc.State state) {
-            if (!state.GetArgument(0, "T").TryGetValueAsString(out var text)) {
+            if (!state.GetArgument(0, "T").TryGetValueAsString(out var text) || text.Length == 0) {
```

Comment:
Strings that are just whitespace also return null
```suggestion
            if (!state.GetArgument(0, "T").TryGetValueAsString(out var text) || string.IsNullOrWhiteSpace(text)) {
```

Reply:
This is already covered because `TryGetTreeEntry` or something returns false. It just throws some exception on empty string. But can't be hurt to have a bit of defensive programming I guess.